docs/contributing.md

```diff
@@ -60,52 +52,44 @@ In order to be eligible for merging, all branches must pass testing and linting
   - For an optimal developer experience, it's recommended that you configure your editor to run linting/formatting inline.
 - All tests must pass, and eslint is run on all files in CircleCI
 
-### Releasing
-
-Steps for a new release (these should be automated as much as possible):
+### Merge to develop
 
-1. Determine the new version number based on the scale of changes following [Semantic Versioning](https://semver.org/):
+When your branch is ready for review, open a PR into `develop` and request reviews from relevant team members. Reviews from codeowners will automatically be requested. Address any failing tests, lint errors, PR feedback, etc., and once your branch is approved you can squash & merge it into `develop`.
 
-> Given a version number MAJOR.MINOR.PATCH, increment the:
->
-> - MAJOR version when you make incompatible API changes,
-> - MINOR version when you add functionality in a backwards compatible manner, and
-> - PATCH version when you make backwards compatible bug fixes.
+Passing `develop` builds will automatically be published to the `next` tag on NPM. This allows users to easily test out the latest version in their applications, which may be unstable.
 
-2. Create the release candidate branch (note that version numbers should _never_ be prefixed with `v`):
+### Releasing
```

Comment:
I thought the point of this change was to make every merge to develop trigger a release based on the merge commit message.

These new docs sound like we still want to follow a manual release process...?